creators/accra/trips_creator_accra.py

```diff
@@ -57,7 +56,7 @@ def add_trips_to_schedule(self, schedule, data):
                     if not ROUTE_FREQUENCY > 0:
                         print("frequency is invalid for route_master " + str(line.id))
                         ROUTE_FREQUENCY = DEFAULT_ROUTE_FREQUENCY
-                except Exception as e:
+                except ValueError as e:
```

Comment:
You are right, this is a bad pattern. But you also need to catch TypeError exception